main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -613,7 +613,15 @@ object CompletionProvider {
     else {
       return None
     }
-    val (completion, _) = enm.cases.foldLeft("", 1)({
+    val (completion, _) = enm.cases.toList.sortWith{
```

Comment:
SourceLocations are already ordered. I think that can be reused out of the box. (e.g. `sortBy(_.loc)`)

Reply:
That is correct. I missed that implementation.